.buildkite/pipeline.yml

```diff
@@ -94,6 +94,28 @@ steps:
       - github_commit_status:
           context: iOS Unit Tests
 
+  - label: React Native Editor Android Tests
+    command: |
+      npm ci --prefer-offline --no-audit --no-progress
+      cd gutenberg/packages/react-native-editor/android
+      ./gradlew testDebug
+    plugins:
+      - *git-cache-plugin
+      - *nvm_plugin
+    agents:
+      queue: android-staging
+
+  - label: React Native Bridge Android Tests
+    command: |
+      npm ci --prefer-offline --no-audit --no-progress
```

Comment:
Similar to https://github.com/wordpress-mobile/gutenberg-mobile/pull/6320/files#r1428541521